src/components/layouts/Settings.tsx

```diff
@@ -70,6 +79,33 @@ export function Settings(props: Props) {
     >
       <div className="grid grid-cols-12 lg:gap-10">
         <div className="col-span-12 lg:col-span-3">
+          {isGroupSettingsActive && (
+            <div
+              className="flex items-center justify-between bg-white border border-gray-200 py-3 rounded space-x-3 px-2"
+              style={{
+                backgroundColor: colors.$1,
+              }}
+            >
+              <div className="flex items-center space-x-2 flex-1 min-w-0">
+                <div>
+                  <Icon element={FaObjectGroup} size={20} />
+                </div>
+
+                <span className="text-sm truncate dark:text-gray-100">
```

Comment:
Note, we're doing styles with color schemes now, please don't add any specific font or background colors.